code/modules/fishing/fish/fish_evolution.dm

```diff
@@ -40,24 +40,44 @@ GLOBAL_LIST_EMPTY(fishes_by_fish_evolution)
 		name = full_capitalize(initial(new_fish_type.name))
 /**
  * The main proc that checks whether this can happen or not.
- * Please do keep in mind a mate may not be present for fish with the
- * self-reproductive trait.
+ * Keep in mind the mate and aquarium arguments may be null if
+ * the fish is self-reproducing or this evolution is a result of a fish_growth component
  */
 /datum/fish_evolution/proc/check_conditions(obj/item/fish/source, obj/item/fish/mate, obj/structure/aquarium/aquarium)
 	SHOULD_CALL_PARENT(TRUE)
 	//chances are halved if only one parent has this evolution.
```

Comment:
Nitpick but you probably also want to move this comment

Reply:
Where to?

Reply:
Just inside the if statement probably? So that it's commenting the line it describes, not the line above the line it describes.